docs/glossary.rst

```diff
@@ -0,0 +1,55 @@
+Glossary
+========
+
+.. glossary::
+
+    authenticator
+        A software or hardware implementation of the client side of the Web Authentication standard. The browser usually presents the user with multiple options when registering or logging in with a credential. The authenticator handles the cryptographic aspects of the process.
+
+    credential ID
+        A unique identifier generated by the authenticator when creating a WebAuthn credential. Credential ID consists of a random sequence of bytes and is used in negotiations between the browser and the server.
+
+    cryptography
+        The practice of securing data through encryption and digital signatures. In the context of WebAuthn, cryptography ensures that authentication is secure and resistant to attacks.
+
+    discoverable credential
+        An alternative term for a passkey or resident key. It refers to a WebAuthn credential that allows authentication without the server providing credential IDs.
+
+    key pair
+        A public-private pair of cryptographic keys used for secure communication and authentication. The public key can be shared freely, while the private key is kept secret and used to decrypt data or create a digital signature.
+
+        The two keys are mathematically linked. So data encrypted with the public key can only be decrypted by the corresponding private key, and vice versa. This ensures secure interactions without needing to share the private key.
+
+    passkey
+        A type of WebAuthn credential that supports passwordless logins in scenarios where the server has not first identified the user and has not provided the web browser with credential IDs.
+
+        While passkeys are WebAuthn credentials, WebAuthn credentials are not always passkeys.
```

Comment:
Still feel like this and discoverable credential should be merged as exchangeable terms - for the record